src/js/components/Router.jsx

```diff
@@ -32,6 +32,11 @@ const AsyncStockMovementInbound = Loadable({
   loading: Loading,
 });
 
+const AsyncStockMovementInbound2 = Loadable({
+  loader: () => import('components/stock-movement-wizard/inboundV2/InboundV2'),
```

Comment:
I don't know whether we should call it V2. If we can quickly change the naming after removing the old inbound I am fine with it. 
What do you think @awalkowiak?

Reply:
It was my suggestion so you can blame me for that. The idea was to just have both of the versions for now, so that Sebastian can quickly look into the old workflow, but it is supposed to be changed at the end of the development, so when the old workflow will be removed.
This is also why I decided to create a feature branch for that.